### ISSUE TITLE
Doesn't open and crashes

### ISSUE THREAD

**Issue by LTUNF** [zhuowei/MCPELauncher#1082]
Everytime I try to open Block Launcher, the screen turns white and a few seconds later, it crashes. Please fix this.

**Comment by DanLeblanc87** [comment]
Yeah. I'm having the same problem myself. Using LG SYTLO and had no problems before until now.

**Comment by zedman2000** [comment]
If you updated to 0.15:
It has not been updated for 0.15.  It takes time to re-hook into all the functions.  It was stated on Twitter that the author is going to TRY to get it ported for 0.15 by the end of the week.  Please close this issue as it is wasting the author's time.  Instead of porting the program he is spending his time closing issue like this which are just fodder.  Porting the program is going to happen, so again, please close this so you do not waste the author's time.
